### DISCUSSION TITLE
How to manually apply "suggestions"?

### DISCUSSION THREAD

**Discussion by Vexs** [sn4k3/UVtools#475] [Q&A] [answered]
I've been using the elephant's foot presentation via rest script for some time, and was excited to see it integrated into the mainline in recent releases- that said, I have no idea how to trigger it's application! The only place I can find mention of it is in the suggestions options menu, and there doesn't appear to be a way to apply it (or other "suggestions") to the file. The "apply when" fields are also a bit difficult to understand to my mind- I have no idea what they do.

**Comment by sn4k3** [comment, marked as answer]
To apply suggestions you click/select on which suggestions you want to apply within the list, and click "Apply" button. Or right click each and click apply. The suggestion you are looking for is the "Wait time before cure".

"apply when" options are actually clear, it let you opt when suggestion will trigger/show/can apply:

- Values are outside limits, which are set by you on configuration under "Limits"
- Values are diferent from the recommended value, the recommend value is calculated based on your configurations

Any optimal value outside the chosen criteria will trigger the suggestion, otherwise they will show as applied.

**Comment by Vexs** [comment]
Ah, I see, it has to be outside those limits before it's possible to apply it. Got it.
I guess for my usecases it's best to set those limits to something silly so I can just manually apply it on-demand. 

Is there any documentation about how these recommended values are created out of curiosity? (source link would also work)

**Reply by sn4k3** [reply]
You can always set apply when value is diferent so limits doesn't matter for that equation. Don't use unreasonable limits just because of that, since limits are there for a porpouse.

> Is there any documentation about how these recommended values are created out of curiosity? (source link would also work)

You mean the final value to be applied to the wait time? Or all the default values in the form?
Either way they are based on Jan study, but people should configure to what they find better. If you right click the suggestion you can go to "More information" and will open the study.

**Reply by Vexs** [reply]
> You can always set apply when value is diferent so limits doesn't matter for that equation. Don't use unreasonable limits just because of that, since limits are there for a porpouse.

Yeah, I figured this out after playing with it. Thanks though

> You mean the final value to be applied to the wait time? Or all the default values in the form?

I see- I guess I misinterpreted and assumed there was some calculation going on but if it's just hard defaults that makes sense.
